docs/clients/grpc/projections/README.md

```diff
@@ -0,0 +1,665 @@
+// This is currently not available in the Rust client
+```
+</xode-block>
+</xode-group>
+
+## List continuous projections
+
+Returns a list of all continuous projections
+
+<xode-group>
+<xode-block title="C#">
+
+<<< @/docs/clients/dotnet/generated/main/samples/projection-management/Program.cs#ListContinuous
+
+</xode-block>
+<xode-block title="Java">
+
+```java
+// This is currently not available in the java client
+```
+</xode-block>
+<xode-block title="NodeJS">
+
+```ts
+listContinuousProjections(options?: ListProjectionsOptions): Promise<ProjectionDetails[]>;
+```
+</xode-block>
+<xode-block title="Rust">
+
+```Rust
+// This is currently not available in the Rust client
+```
+</xode-block>
+</xode-group>
+
+## Get Status 
+
+Gets the status of a named projection
+
+<xode-group>
+<xode-block title="C#">
+
+<<< @/docs/clients/dotnet/generated/main/samples/projection-management/Program.cs#GetStatus
+
+</xode-block>
+<xode-block title="Java">
+
+```java
+// This is currently not available in the java client
+```
+</xode-block>
+<xode-block title="NodeJS">
+
+```ts
+ getProjectionStatistics(projectionName: string,options?: GetProjectionStatisticsOptions): Promise<ProjectionDetails>;
+```
+</xode-block>
+<xode-block title="Rust">
+
+```Rust
+// This is currently not available in the Rust client
+```
+</xode-block>
+</xode-group>
+
+## Get state
+
+Retrieves the state of a projection.
+
+<xode-group>
+<xode-block title="C#">
+
+```csharp
+public async Task<JsonDocument> GetStateAsync(string name, string? partition = null, UserCredentials? userCredentials = null, CancellationToken cancellationToken = default(CancellationToken))
+public async Task<T> GetStateAsync<T>(string name, string? partition = null, JsonSerializerOptions? serializerOptions = null, UserCredentials? userCredentials = null, CancellationToken cancellationToken = default(CancellationToken))
+
+```
+</xode-block>
+<xode-block title="Java">
+
+```java
+// This is currently not available in the java client
+```
+</xode-block>
+<xode-block title="NodeJS">
+
+```ts
+getProjectionState<T = unknown>(projectionName: string,options?: GetProjectionStateOptions): Promise<T>;
+```
+</xode-block>
+<xode-block title="Rust">
+
+```Rust
+// This is currently not available in the Rust client
+```
+</xode-block>
+</xode-group>
+
+## Get result
+
+Retrieves the result of the named projection and partition.
+
+<xode-group>
+<xode-block title="C#">
+
+<<< @/docs/clients/dotnet/generated/main/samples/projection-management/Program.cs#GetResult
+
+</xode-block>
+<xode-block title="Java">
+
+```java
+public <TResult> CompletableFuture<TResult> getResult(final String projectionName, Class<TResult> type)
+public <TResult> CompletableFuture<TResult> getResult(final String projectionName,  Function<TypeFactory, JavaType> javaTypeFunction)
+
+```
+</xode-block>
+<xode-block title="NodeJS">
+
+```ts
+getProjectionResult<T = unknown>(projectionName: string, options?: GetProjectionResultOptions): Promise<T>
```

Comment:
Similar comment as https://github.com/EventStore/documentation/pull/404/files#r689297932 regarding the API Reference vs snippets.

Reply:
same as above, this needs a decision .